Makefile

```diff
@@ -29,6 +29,18 @@ test:
 	docker-compose up -d db queue
 	docker-compose -f docker-compose.test.yml up --build --force-recreate --exit-code-from unit_test unit_test
 
+postman:
+	# This target should be executed by passing in an argument for the environment (dev/test/sbx)
+	# and if needed a token.
+	# For example: make postman env=test token=<MY_TOKEN>
+	docker-compose -f docker-compose.test.yml run postman_test test/$(env).postman_environment.json --global-var "token=$(token)"
+
+
+postman-local:
```

Comment:
Do we need two postman targets?  Might get confusing as to which one to use in which situation. Can we get away with just one?

Reply:
One spins up the local environment and the other one doesn't.  I don't see any reason to spin that up if we are just testing against an external source.  I will add a comment so that it is more clear.